src/pages/groupComparison/MultipleCategoryBarPlot.tsx

```diff
@@ -425,6 +430,43 @@ export default class MultipleCategoryBarPlot extends React.Component<
 
     @computed get labels() {
         if (this.data.length > 0) {
+            if (this.props.sortByOption == 'SortByTotalSum') {
+                const majorCategoryCounts: any = {};
+
+                this.data.forEach(item => {
```

Comment:
this code looks pretty similar to the the function `sortDataByOption` you made below. it would be nice to extract this into a function that can be reused